packages/relay/src/utils.ts

```diff
@@ -67,4 +67,42 @@ export class Utils {
   public static generateRequestId = (): string => {
     return crypto.randomUUID();
   };
+
+  /**
+   * Estimates the transaction fees for file create and file append transactions based on the provided
+   * call data size, file chunk size, and current network exchange rate.
+   *
+   * @param {number} callDataSize - The size of the call data in bytes.
+   * @param {number} fileChunkSize - The size of each file chunk in bytes.
+   * @param {number} currentNetworkExchangeRateInCents - The current network exchange rate in cents.
+   * @returns {number} The estimated transaction fee in tinybars.
+   */
+  public static estimateFileTransactionsFee(
+    callDataSize: number,
+    fileChunkSize: number,
+    currentNetworkExchangeRateInCents: number,
+  ): number {
+    const fileCreateTransactions = 1;
+    const fileCreateFeeInCents = constants.NETWORK_FEES_IN_CENTS.FILE_CREATE_PER_5_KB;
+
```

Comment:
Do we need to validate that `fileChunkSize` is not a zero value to prevent divide by zero issues in the next line?
```
  if (fileChunkSize <= 0) {
    throw new Error('fileChunkSize must be greater than zero.');
  }
  ```